libs/libc/misc/Kconfig

```diff
@@ -61,3 +61,9 @@ config LIBC_ENVPATH
 	---help---
 		Use the contents of the common environment variable to locate executable
 		or library files.  Default: n
+
+config LIBC_ERR
+	bool "Support err() verr() verrx() warn() vwarn() vwarnx()"
+	default n
```

Comment:
Would it be better to enable it by default and then, any prior user can opt out?